src/services/importService.ts

```diff
@@ -158,4 +187,74 @@ export default class ImportService implements IImportService {
         }
         return metadata;
     }
+
+    /**
+     * Generate assets for V1 Image Project frames and regions
+     * @param v1Project - v1 Project content and file information
+     * @param frameList - Dictionary of frames:regions in v1 project
+     * @param assetService - assetService corresponding to v2 project
+     */
+    private async generateImageAssets(v1Project: IFileInfo, frameList: {[frameName: string]: IV1Region[]},
+                                      assetService: AssetService): Promise<IAssetMetadata[]> {
+        const generatedAssetMetadata: IAssetMetadata[] = [];
+        const originalProject = JSON.parse(v1Project.content as string);
+
+        for (const frameName in frameList) {
```

Comment:
Consider breaking inner loop logic into its own function to make this easier to read / maintain.